### ISSUE TITLE
suggestion: compatibility with tinkers construct

### ISSUE THREAD

**Issue by RCXcrafter** [diesieben07/Modjam-4#9]
I think it would be cool if you could make the tool materials from this mod usable tool materials for tinkers construct.

In case you didn't know, they have an easy to use API:
https://gist.github.com/bonii-xx/e46f9d9e81e29d796b1b

**Comment by DPxKnightxZero** [comment]
Yeah this needs TiC before I add it however I will keep an eye on this mod.
